### ISSUE TITLE
a11y: Form Controls relies on colour to convey invalid meaning

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#342]
# Platform

- Web

# Description

https://compound.element.io/?path=/story/compound-web_form-controls-action--invalid uses colour alone to convey the input is invalid - this is not accessible

Ref https://element-io.atlassian.net/browse/PSB-968

<img width="720" height="720" alt="Image" src="https://github.com/user-attachments/assets/4c90f28a-1835-4dc3-97bc-f86b83eaeded" />

<img width="720" height="720" alt="Image" src="https://github.com/user-attachments/assets/8f2f86ed-6305-4287-a112-be7b152a5731" />

<img width="720" height="720" alt="Image" src="https://github.com/user-attachments/assets/9b8b35d3-6774-499f-af88-4ec262d69e85" />
